web/wwwroot/css/site.css

```
@@ -72,17 +161,38 @@ html {
 }
 
 #mainLayoutHeaderSections .selectedTopNav {
     background-color: #ffe599; /* ucdavis-gold-40 */
 }
 
+/* Quasar's overlay-drawer glow bleeds a stray line down the left edge. */
+.q-drawer--mobile .q-layout__shadow:after {
+    box-shadow: none;
+}
+
 /*Left Nav styles*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)